src/events/interactionCreate.ts

```diff
@@ -0,0 +1,8 @@
+import { Client, Events, Interaction } from "discord.js";
+import commandHandler from "../commandHandler";
+
+export default (client: Client): void => {
+    client.on(Events.InteractionCreate, async(interaction: Interaction) => {
+        await commandHandler(interaction);
```

Comment:
Then we can do type guarding here, and properly cast to, and allow for multiple types of interactions (buttons, etc)

Reply:
See above comment